sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -41,18 +39,20 @@ public static DirectConnectionConfig getDefaultConfig() {
         return new DirectConnectionConfig();
     }
 
-    //  TODO: (DANOBLE) - To update these docs.
-
     /**
-     * Gets the direct connection timeout
+     * Gets the connection timeout for direct client,
+     * represents timeout for establishing connections with an endpoint
+     *
```

Comment:
Guidance on when to tune to what values will help CX.

Reply:
Guidance on these can vary based on customer, case to case. Referring material in perf guide will help and as you mentioned we can refer the perf guide in this docs.